### ISSUE TITLE
The titles in the data view sections should be bold text

### ISSUE THREAD

**Issue by dancingsushii** [amosproj/amos2024ss04-building-information-enhancer#248]
## User story
1. As a user,
2. I want to see titles in the data view section in bold,
3. So that I can see important things better.

## Acceptance criteria
* All titles in the data view sections are displayed in bold text.
* The bold styling does not affect the readability or layout of other text and elements within the data view.
* Ensure that the bold titles are consistent across all browsers.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
